core/src/display_object/movie_clip.rs

```diff
@@ -1120,8 +1118,8 @@ impl<'gc> MovieClip<'gc> {
         depth: Depth,
         place_object: &swf::PlaceObject,
     ) -> Option<DisplayObject<'gc>> {
-        let library = context.library.library_for_movie_mut(self.movie().unwrap()); //TODO
-        let avm_type = library.avm_type();
+        let movie = self.movie().unwrap();
```

Comment:
do you know what that  `TODO` was there about?

Reply:
No idea. That's why I decided to remove it.